### DISCUSSION TITLE
EP messed up my W11 big time! Need help!

### DISCUSSION THREAD

**Discussion by arniep** [valinet/ExplorerPatcher#2826] [Q&A]
Just installed the latest version on EP but this app has really messed up my W11 pro HP computer. When I first installed it yesterday, it seemed to work as advertised. However, when I went to use it today, it was terrible... I set it up to imitate W10 file explorer and start menu and initially it seemed to work great yesterday. When I went to use it today, without having changed anyting on my PC, left-clicking on the start button caused a few seconds on the spinning wheel and then nothing. No start menu appeared. Right-clicking on the Start button did open the usual utilities menu, but clicking on Settings was problematic. It did open the Settings menu but clicking on any of the settings categories dumped immediately back to the desktop. The only item that worked was the shutdown/restart button.
So I thought I would try to uninstall ep_setup.exe. I found a note on their github to rename the app to ep_uninstall.exe which I did. That gave me a success message and I then rebooted. ALthough I now have the W11 start menu button behavior back, I still cannot open anything within Settings without still getting dumped right back to the desktop. Apparently, Explorer Patcher even changed my trackpad to click only, no tapping and cannot even get back into the trackpad/mouse settings now.
Help!

**Comment by pyrates999** [comment]
already being discussed here: https://github.com/valinet/ExplorerPatcher/issues/2822
